src/main/java/dev/ebullient/convert/tools/dnd5e/Json2QuteFeat.java

```diff
@@ -25,15 +29,18 @@ protected Tools5eQuteBase buildQuteResource() {
         List<String> text = getFluff(Tools5eIndexType.featFluff, "##", images);
         appendToText(text, SourceField.entries.getFrom(rootNode), "##");
 
-        // TODO: update w/ category, ability, additionalSpells
+        JsonNode abilityNode = FeatFields.ability.getFrom(rootNode);
+
+        // TODO: update w/ category, additionalSpells
         return new QuteFeat(sources,
                 type.decoratedName(rootNode),
                 getSourceText(sources),
                 listPrerequisites(rootNode),
                 null, // Level coming someday..
                 images,
                 String.join("\n", text),
-                tags);
+                tags,
+                getAbilityScoreIncreases(abilityNode));
```

Comment:
I would prefer these arguments in a different order (convention/predictability). See comments in QuteFeat below